rbac/common/role/propose_member.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2018 Contributors to Hyperledger Sawtooth
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# -----------------------------------------------------------------------------
+
+import logging
+from uuid import uuid4
+from rbac.addressing import addresser
+from rbac.common import protobuf
+from rbac.common.manager.base_message import BaseMessage
+
+LOGGER = logging.getLogger(__name__)
+
+
+class ProposeAddRoleMember(BaseMessage):
+    def __init__(self):
+        BaseMessage.__init__(self)
+
+    @property
+    def name(self):
+        return "proposal"
```

Comment:
What is the purpose of this block of code? I see it duplicated across proposals, rejections... invoking name() on any of them will yield the same result (?)

Reply:
It's so the base class knows the identifier and container addresses.